lighthouse-core/lib/lh-error.js

```diff
@@ -15,10 +15,15 @@ const UIStrings = {
    * */
   didntCollectScreenshots: `Chrome didn't collect any screenshots during the page load. Please make sure there is content visible on the page, and then try re-running Lighthouse. ({errorCode})`,
   /**
-   * @description Error message explaining that the network trace was not able to be recorded for the Lighthouse run.
+   * @description Error message explaining that the performance trace was not able to be recorded for the Lighthouse run.
    * @example {NO_TRACING_STARTED} errorCode
    * */
   badTraceRecording: 'Something went wrong with recording the trace over your page load. Please run Lighthouse again. ({errorCode})',
+  /**
+   * @description Error message explaining that the First Contentful Paint metric was not seen during the page load.
+   * @example {NO_FCP} errorCode
+   * */
+  noFcp: 'The page did not paint any content. Please ensure you keep the browser window foreground during the load and try again. ({errorCode})',
```

Comment:
shoot sorry I missed this, `in the` or `foregrounded`?

```suggestion
  noFcp: 'The page did not paint any content. Please ensure you keep the browser window in the foreground during the load and try again. ({errorCode})',
```